vectorio/_rectangle.py

```diff
@@ -51,7 +52,7 @@ def __init__(
         """
         self._width = 1
         self._height = 1
-        self._color_index = 1
+        self._color_index = color_index
```

Comment:
I think we need to `+1` here in order to match the core implementation. https://github.com/adafruit/circuitpython/blob/main/shared-module/vectorio/Rectangle.c#L18